portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue

```diff
@@ -557,7 +557,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         position: absolute;
         left: -20px;
         z-index: 1;
-        content: "YES";
+        content: "";
```

Comment:
Is the content rule still needed ?

Reply:
yes , when we remove it the content will not displayed